pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight"
-version = "0.11.10"
+version = "0.11.11"
```

Comment:
I'd say that since it isn't just a bug fix but adds a new check we could increment the minor version, e.g. 0.12.0 . But if others disagree i don't mind keeping it this way.

Reply:
I would agree with that - we should definitely try to get in the habit of minor version increments when functionality is added.